rules/coding-style/src/Rector/Catch_/CatchExceptionNameMatchingTypeRector.php

```diff
@@ -12,6 +13,7 @@
 
 /**
  * @see \Rector\CodingStyle\Tests\Rector\Catch_\CatchExceptionNameMatchingTypeRector\CatchExceptionNameMatchingTypeRectorTest
+ * @see https://regex101.com/r/xmfMAX/1
```

Comment:
Great! Just now it's not clear what the link belongs to. 

This is the best example for location:

https://github.com/rectorphp/rector/blob/d5dea26151a701aab2fd931e3223aef4e9e7c252/src/PhpDoc/PhpDocTagsFinder.php#L14-L21

Could you mimic it = add a constant with link right above the constants?